functests/1_performance/performance.go

```diff
@@ -360,15 +362,26 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			By("Checking v1alpha1 -> v1 conversion")
 			v1alpha1Profile.Name = "v1alpha"
 			v1alpha1Profile.ResourceVersion = ""
+			v1alpha1Profile.Spec.NodeSelector = map[string]string{"test": "test"}
```

Comment:
Why do we need this?

Reply:
in general, we should have validation that should check that we do not have two profile with the same node selector(but I failed to find it under the code, so I think we need to add it), but anyway I want to avoid the creation of duplicate MC and KC for the same MCP because it can start update flow for the MCP

Reply:
@cynepco3hahue please file a PAO issue about the missing validation